scripts/ci/libraries/_kind.sh

```diff
@@ -69,7 +69,7 @@ function kind::make_sure_kubernetes_tools_are_installed() {
     kubectl_url="https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/${system}/${machine}/kubectl"
     if [[ -f "${KUBECTL_BINARY_PATH}" ]]; then
         local downloaded_kubectl_version
-        downloaded_kubectl_version="$(${KUBECTL_BINARY_PATH} version --client=true --short | awk '{ print $3 }')"
```

Comment:
Why no `--short`?

Reply:
We will be re-doing this part when we add it to Breeze (I hope shortly). I will respond then ok :)?